app/controllers/batch_controller.rb

```diff
@@ -14,8 +14,8 @@ def default_values
     {
       "creator" => [current_user.name],
       "title" => @batch.generic_files.map(&:label),
-      "publisher" => [I18n.t('form_defaults.publisher')],
-      "language" => [I18n.t('form_defaults.language_uri')]
+      "publisher" => [TriplePoweredResource.new(RDF::URI(I18n.t('form_defaults.publisher')))],
+      "language" => [I18n.t(TriplePoweredResource.new(RDF::URI(I18n.t('form_defaults.language_uri'))), :default => 'English')]
```

Comment:
If you save this with English as the default, is it saving the string as the default or the URI?